technical_liaisons.md

```diff
@@ -0,0 +1,25 @@
+# Technical Liaisons
+
+DLS will designate a technical liaison to Each project in its portfolio.
+
+## Why Technical Liaisons
+
+* Ensure that each of our projects are stewarded, looked over, and thought about reliably.
+* Empower developers to learn deeply about the context and stakeholder use cases for a particular application.
+* Act as a point of contact for the product owner to support interactions with stakeholders.
+* Bring knowledge of issues and talking points back to the team for implementation discussion and prioritization.
+* Promote the needs of their application to the team to ensure it isn't lost in the shuffle.
```

Comment:
Suggestion: Maybe something along the lines of `Product owners have told us this role is useful`? To point towards the fact that we've asked.

Reply:
I sort of think this is implied by line 9. I guess it doesn't hurt to add it to that point.